hack/util.sh

```diff
@@ -71,18 +71,26 @@ function configure_os_server {
 # start_os_server starts the OS server, exports the PID of the OS server
 # and waits until OS server endpoints are available
 function start_os_server {
-  echo "[INFO] Starting OpenShift server"
-  sudo env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
-    --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
-    --node-config=${NODE_CONFIG_DIR}/node-config.yaml \
-    --loglevel=4 \
-    &> "${BASETMPDIR}/openshift.log" &
-  export OS_PID=$!
-
-  wait_for_url "${KUBELET_SCHEME}://${KUBELET_HOST}:${KUBELET_PORT}/healthz" "[INFO] kubelet: " 0.5 60
-  wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 80
-  wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 80
-  wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/api/v1/nodes/${KUBELET_HOST}" "apiserver(nodes): " 0.25 80
+    echo "[INFO] Scan of OpenShift related processes already up via ps -ef  | grep openshift : "
+    ps -ef | grep openshift
+    echo "[INFO] Starting OpenShift server"
+    sudo env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
+	 --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
+	 --node-config=${NODE_CONFIG_DIR}/node-config.yaml \
+	 --loglevel=4 \
+    &> "${LOG_DIR}/openshift.log" &
+    export OS_PID=$!
+
+    echo "[INFO] opnshft svr start at: "
+    echo `date`
+  
+    wait_for_url "${KUBELET_SCHEME}://${KUBELET_HOST}:${KUBELET_PORT}/healthz" "[INFO] kubelet: " 0.5 60
+    wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 80
+    wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 80
+    wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/api/v1/nodes/${KUBELET_HOST}" "apiserver(nodes): " 0.25 80
+    
+    echo "[INFO] opnshft svr start healtch checks done at: "
```

Comment:
minus the "healtch", I was intentionally using short hand ;-) ; however, my mindset was "this is a test and hence internal", but alas, our tests are open to the world as well, aren't they ? ;-) ... old habits die hard ... will address in next push